od/2.1/es/index.markdown

```diff
@@ -0,0 +1,124 @@
+---
+title: Definición de Conocimiento Abierto
+layout: page
+---
+
+Versión: 2.1
+
+La Definición de Conocimiento Abierto aporta precisión al significado del término «abierto» (open) cuando se aplica al conocimiento, promueve un procomún robusto en el que cualquiera puede participar y maximiza su interoperabilidad.
+
+**Resumen:** *El conocimiento es abierto si cualquiera es libre de acceder a él, usarlo, modificarlo y compartirlo, estando sujeto a lo sumo a medidas que preserven su autoría y su apertura.*
+
+Este significado esencial se corresponde con el de «abierto» (open) cuando se refiere al software, como en la [Open Source Definition](http://opensource.org/docs/osd1), y es un sinónimo de «libre» (free) tal y como se usa en la [Definición de Software Libre](https://en.wikipedia.org/wiki/The_Free_Software_Definition) y en la [Definición de Obras Culturales libres](https://en.wikipedia.org/wiki/Definition_of_Free_Cultural_Works) .
```

Comment:
The OSD link is broken. Maybe the trailing '1'? In any case I suggest using available eswp links:

- https://es.wikipedia.org/wiki/Open_Source_Definition
- https://es.wikipedia.org/wiki/Definici%C3%B3n_de_Software_Libre
- https://es.wikipedia.org/wiki/Definici%C3%B3n_de_obras_culturales_libres

Reply:
Done!